src/components/InfoNote/index.js

```diff
@@ -5,11 +5,9 @@ import Message from '../Message';
 
 function InfoNote({ children }) {
```

Comment:
I think we can delete this component now. Since using it or using the Message component with the `info` theme is the same thing now.